SUPPORTED_APIS.md

```diff
@@ -2,31 +2,63 @@
 
 > ⚠️ **WORK IN PROGRESS**: This list is non-comprehensive and being updated
 
-# Key
+## Supported APIs Table
+
+| Category | API | Status | Description |
+| --- | --- | --- | --- |
+| [`CONFIG NAMESPACE`](#config-namespace) | [`config_getShowCalls`](#config_getShowCalls) | SUPPORTED | Gets the current value of `show_calls` that's originally set with `--show-calls` option |
+| [`CONFIG NAMESPACE`](#config-namespace) | [`config_setShowCalls`](#config_setShowCalls) | SUPPORTED | Updates `show_calls` to print more detailed call traces |
+| [`CONFIG NAMESPACE`](#config-namespace) | [`config_setResolveHashes`](#config_setResolveHashes) | SUPPORTED | Updates `resolve-hashes` to call OpenChain for human-readable ABI names in call traces |
+| [`NETWORK METHODS`](#network-methods) | [`net_getVersion`](#net-getVersion) | NOT IMPLEMENTED | Returns the current network id |
+| [`NETWORK METHODS`](#network-methods) | [`net_getMetadata`](#net-getMetadata) | NOT IMPLEMENTED | Returns the current network metadata |
```

Comment:
The only `NET` namespace APIs I know of (https://ganache.dev/):

1. `net_version`
2. `net_peerCount`
3. `net_listening`

And we've implemented them here https://github.com/matter-labs/era-test-node/blob/main/src/net.rs

Reply:
Awesome will include these